plone/app/locales/locales/nl/LC_MESSAGES/widgets.po

```diff
@@ -119,31 +123,31 @@ msgstr "Bladeren"
 
 #: ./patterns/resourceregistry/js/registry.js
 msgid "Build"
-msgstr ""
+msgstr "Bouwen"
 
 #: ./patterns/thememapper/pattern.js
 msgid "Build CSS"
-msgstr ""
+msgstr "Bouw CSS"
 
 #: ./patterns/resourceregistry/js/builder.js
 msgid "Build it"
-msgstr ""
+msgstr "Bouw het"
```

Comment:
Engelse brontekst is beetje vreemd: niet gewoon 'build' zoals bij de registry.js hierboven?

Reply:
widgets.pot is outdated. This pattern does not exist anymore. I have opened https://github.com/collective/plone.app.locales/issues/358